Clinical-Genomics/scout
# Patch release 4.30.1 - Fix broken cryptography dependency

Fix #2419 . 
See original issue here: https://github.com/pyca/cryptography/issues/5771

**How to test**:
1. On master branch, reproduce the error by running `docker build -t scout .`
1. Switch to this branch and make sure that the same command works

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
